pkg/config/config.go

```diff
@@ -149,6 +153,8 @@ func (c *Config) RegisterFlags(f *flag.FlagSet) {
 
 	f.StringVar(&c.ReloadAddress, "reload-addr", "127.0.0.1", "address to expose a secondary server for /-/reload on.")
 	f.IntVar(&c.ReloadPort, "reload-port", 0, "port to expose a secondary server for /-/reload on. 0 disables secondary server.")
+	f.StringVar(&c.BasicAuthUser, "basic-auth-user", "", "basic auth username for fetching remote config.")
+	f.StringVar(&c.BasicAuthPassFile, "basic-auth-password-file", "", "path to file containing basic auth password for fetching remote config.")
```

Comment:
I think the flag names should be namespaced to make it clearer what they're used for (i.e., `config.url.basic-auth-user`)